### PR TITLE
Gives pais the ability to toot

### PR DESCRIPTION
**Pull request by anconfuzedrock**
:cl: Confused Rock
rscadd: Pais can now install the loudness booster to play horrible music so they can sound and look awful at the same time!
/:cl:
This allows PAIs to play a recorder (if it were a violin or something it'd probably sound really annoying) if they download the proper software. I think this is the kind of mostly useless feature that pais are built for, and that's about it.

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
well done lad

File: code/modules/mob/living/silicon/pai/software.dm [hunk 1]
```diff
@@ -64,6 +65,8 @@
 				left_part = softwareCamera()
 			if("signaller")
 				left_part = softwareSignal()
+			if("loudness")
```
Comment (AnturK) [inline code/modules/mob/living/silicon/pai/software.dm]: I know you're just following the convention here but man these are ugly.

File: code/modules/mob/living/silicon/pai/pai.dm [hunk 2]
```diff
@@ -12,6 +12,7 @@
 	maxHealth = 500
 	layer = BELOW_MOB_LAYER
 	can_be_held = TRUE
+	var/obj/item/instrument/recorder/P
```
Comment (AnturK) [inline code/modules/mob/living/silicon/pai/pai.dm]: Name this something more relevant please. Like internal_instrument or whatever.

**Review by AnturK — CHANGES_REQUESTED** [review]
You need to add deletion of the instrument on pAI destroy.

**Comment by MrDoomBringer** [comment]
can we get a "good first PR" tag

**Comment by anconfuzedrock** [comment]
Right I'll add the deletion not that it changes anything

**Comment by anconfuzedrock** [comment]
Think this fixes it.

**Comment by anconfuzedrock** [comment]
Am I forgetting something or do I need to somehow resolve the requested changes or what?

**Comment by MrDoomBringer** [comment]
@anconfuzedrock nah fam ur good, just w8 4 merge

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**